src/ProofOfConcepts/ConnectorTypesGallery/ConnectorTypesGalleryListView.tsx

```diff
@@ -0,0 +1,128 @@
+  Label,
+  LabelGroup,
+} from '@patternfly/react-core';
+import OutlinedQuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/outlined-question-circle-icon';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+export type ConnectorTypesListItemProps = {
+  id: string;
+  labels: string[];
+  title: string;
+  verison: string;
+};
+
+export const ConnectorTypeListItem: React.FunctionComponent<ConnectorTypesListItemProps> =
+  ({ id, labels = [], title, verison }) => {
+    const [selectedDataListItemId, setSelectedDataListItemId] =
+      React.useState('');
+
+    const onSelectDataListItem = (id) => {
+      setSelectedDataListItemId(id);
+    };
+
+    const handleInputChange = (
+      id: string,
+      _event: React.FormEvent<HTMLInputElement>
+    ) => {
+      setSelectedDataListItemId(id);
+    };
+
+    const { t } = useTranslation();
+
+    return (
+      <DataList
+        aria-label="selectable data list connector"
+        selectedDataListItemId={selectedDataListItemId}
+        onSelectDataListItem={onSelectDataListItem}
+        selectableRow={{ onChange: handleInputChange }}
+      >
+        <DataListItem aria-labelledby={id} id={id}>
+          <DataListItemRow>
+            <Flex justifyContent={{ default: 'justifyContentFlexStart' }}>
+              <FlexItem>
+                {labels.includes('source') ? (
+                  <PlusCircleIcon size="md" color="blue" />
+                ) : (
+                  <PlusCircleIcon size="md" color="green" />
+                )}
+              </FlexItem>
+              <FlexItem>
+                <DataListItemCells
+                  dataListCells={[
+                    <DataListCell key="primary content">
+                      <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                          <Flex>
+                            <FlexItem>
+                              <strong
+                                className="pf-u-font-weight-bold"
+                                id="selectable-action-item1"
+                              >
+                                {title}
+                              </strong>
+                            </FlexItem>
+                            <FlexItem>
+                              <OutlinedQuestionCircleIcon
+                                href="#filled"
+                                color="grey"
+                              />
+                            </FlexItem>
+                            <FlexItem>
+                              <span className="pf-u-font-size-sm pf-u-color-400">
+                                {verison}
+                              </span>
+                            </FlexItem>
+                          </Flex>
+                        </FlexItem>
+                        <FlexItem>
+                          <LabelGroup numLabels="10">
```

Comment:
I forget, does `LabelGroup` need an accurate count?  Maybe it'd be worth creating a variable to hold the result of the filtering/sorting on line 97 below.

Reply:
The numLabels was to make it match the appearance in the from the ticket, to avoid having the "show less" or "show more". I also applied the other changes mentioned.

Reply:
Ah great!  That link was bugging me in the card variant, I hadn't looked up yet how to get rid of it.  Now I don't have to, thanks!  :-)